### PR TITLE
Samples tests

### PR DESCRIPTION
**Pull request by indebanvdhamer**
## Motivation

After forgetting to change the `react-and-scss` sample in https://github.com/wearereasonablepeople/webpacker/pull/67, we figured some tests would be useful. For now we'll just test if the build doesn't crash / produce any errors. This way we cannot forget to update and reflect breaking changes in the samples.

## Changes
- Update `react-and-scss` sample to conform to current `webpacker` version
- Add tests to samples

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,6 +75,10 @@
     "git-cz": "^1.7.1",
     "jest-cli": "^23.4.2",
     "jest-config": "^23.4.2",
+    "webpacker-cyclejs-sample": "file:./samples/cyclejs",
+    "webpacker-react-and-favicon-sample": "file:./samples/react-and-favicon",
+    "webpacker-react-and-graphql-sample": "file:./samples/react-and-graphql",
+    "webpacker-react-and-sass-sample": "file:./samples/react-and-sass",
```
Comment (renanborgez) [inline package.json]: 🥇

File: tests/samples/testRunner.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
```
Comment (renanborgez) [inline tests/samples/testRunner.js]: missing no-console?

Comment (indebanvdhamer) [reply tests/samples/testRunner.js]: Oops, forgot to remove

File: package.json (hunk 1 above)
Comment (d-eisenga) [reply package.json]: 👏

**Pull request merged**